README.md

```diff
@@ -75,16 +75,17 @@ package org.example;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
 
-import com.fauna.annotation.FaunaField;
 import com.fauna.client.Fauna;
 import com.fauna.client.FaunaClient;
-import com.fauna.client.FaunaConfig;
 import com.fauna.exception.FaunaException;
 import com.fauna.query.builder.Query;
 import com.fauna.response.QuerySuccess;
-import com.fauna.serialization.generic.PageOf;
 import com.fauna.types.Page;
 
+import static com.fauna.codec.generic.pageOf;
```

Comment:
This doesn't work for me.
```suggestion
import com.fauna.codec.PageOf;
```

Reply:
oops. it should be `import static com.fauna.codec.Generic.pageOf`;

Reply:
Nice. I updated my suggestion.